ui/pages/onboarding-flow/privacy-settings/privacy-settings.js

```diff
@@ -212,284 +225,455 @@ export default function PrivacySettings() {
-                  {t('ensDomainsSettingDescriptionIntroduction')}
+                <Text variant={TextVariant.headingLg} as="h2">
+                  {selectedItem && selectedItem.title}
                 </Text>
-                <Box
-                  as="ul"
-                  marginTop={4}
-                  marginBottom={4}
-                  paddingInlineStart={4}
-                  style={{ listStyleType: 'circle' }}
-                >
-                  <Text variant={TextVariant.inherit} as="li">
-                    {t('ensDomainsSettingDescriptionPart1')}
-                  </Text>
-                  <Text variant={TextVariant.inherit} as="li">
-                    {t('ensDomainsSettingDescriptionPart2')}
-                  </Text>
-                </Box>
-                <Text variant={TextVariant.inherit}>
-                  {t('ensDomainsSettingDescriptionOutroduction')}
-                </Text>
-              </>
-            }
-          />
-          <Setting
-            value={turnOnCurrencyRateCheck}
-            setValue={setTurnOnCurrencyRateCheck}
-            title={t('currencyRateCheckToggle')}
-            dataTestId="currency-rate-check-toggle"
-            description={t('currencyRateCheckToggleDescription', [
-              <a
-                key="coingecko_link"
-                href={COINGECKO_LINK}
-                rel="noreferrer"
-                target="_blank"
-              >
-                {t('coingecko')}
-              </a>,
-              <a
-                key="cryptocompare_link"
-                href={CRYPTOCOMPARE_LINK}
-                rel="noreferrer"
-                target="_blank"
-              >
-                {t('cryptoCompare')}
-              </a>,
-              <a
-                key="privacy_policy_link"
-                href={PRIVACY_POLICY_LINK}
-                rel="noreferrer"
-                target="_blank"
-              >
-                {t('privacyMsg')}
-              </a>,
-            ])}
-          />
-          <Setting
-            value={turnOnPetnames}
-            setValue={setTurnOnPetnames}
-            title={t('petnamesEnabledToggle')}
-            description={t('petnamesEnabledToggleDescription')}
-          />
-          <ButtonPrimary
-            size={ButtonPrimarySize.Lg}
-            onClick={handleSubmit}
-            block
-            marginTop={6}
-          >
-            {t('done')}
-          </ButtonPrimary>
+              </Box>
+            </Box>
+
+            <div
+              className="privacy-settings__settings"
+              data-testid="privacy-settings-settings"
+            >
+              {selectedItem && selectedItem.id === 1 ? (
+                <>
+                  <Setting
+                    dataTestId="basic-functionality-toggle"
+                    value={externalServicesOnboardingToggleState}
+                    setValue={(toggledValue) => {
+                      if (toggledValue === false) {
```

Comment:
I know it wasn't introduced in this PR but should probably go with the `true` option first.